nuxeo-drive-core/src/test/java/org/nuxeo/drive/service/TestNuxeoDriveManager.java

```diff
@@ -294,6 +294,7 @@ public void testSynchronizationRootDeletion() throws Exception {
 
         // check physical deletion of a parent folder
         session.removeDocument(session.getDocument(new PathRef("/default-domain")).getRef());
+        txFeature.nextTransaction();
         session.save();
```

Comment:
Would you mind checking if `session.save()` is still needed and remove it if not ? I believe it wasn't needed in the first place.